### ISSUE TITLE
"safe init" does not work due to the lack of a .vault-token file

### ISSUE THREAD

**Issue by migs** [starkandwayne/safe#125]
This is a fresh deployment of "safe-boshrelease" (version 0.1.0), using safe 0.6.2

The error:
```
vagrant@bosh-bastion:~$ safe init
!! open /home/vagrant/.vault-token: no such file or directory
```

I can work-around this using the usual `vault init` commands. These create the .vault-token file, and I can then use safe normally.

Should I expect `safe init` to work on a fresh vault/safe-boshrelease deployment, where there are no tokens yet generated due to the vault being uninitialised?

**Comment by jhunt** [comment]
This sounds like a silly oversight on my part; the .vault-token shouldn't be necessary, but is probably being required by the codepaths taken.

What happens if you `touch ~/.vault-token; safe init`?

**Comment by migs** [comment]
```
vagrant@bosh-bastion:~$ curl -k https://10.20.0.20/v1/sys/health
{"initialized":false,"sealed":true,"standby":true,"server_time_utc":1516964473,"version":"0.9.0"}
vagrant@bosh-bastion:~$ safe -k target https://10.20.0.20 $project_id
Now targeting <snip> at https://10.20.0.20
vagrant@bosh-bastion:~$ safe init
!! open /home/vagrant/.vault-token: no such file or directory
vagrant@bosh-bastion:~$ touch /home/vagrant/.vault-token
vagrant@bosh-bastion:~$ safe init
!! no vault token specified; are you authenticated?
vagrant@bosh-bastion:~$ 
```

**Comment by jhunt** [comment]
This should be fixed in 0.6.3
